src/core/net/ip6_address.hpp

```diff
@@ -325,14 +325,25 @@ class Address : public otIp6Address
     /**
      * This method sets the prefix content of Mesh Local Prefix-Based Multicast Address.
      *
-     * @param[in]  aMeshLocalPrefix   A reference to the  Mesh Local Prefix.
+     * @param[in]  aMeshLocalPrefix   A reference to the Mesh Local Prefix.
      *
      */
     void SetMulticastNetworkPrefix(const Mle::MeshLocalPrefix &aMeshLocalPrefix)
     {
         SetMulticastNetworkPrefix(aMeshLocalPrefix.m8, Mle::MeshLocalPrefix::kLength);
     }
 
+    /**
+     * This method sets the prefix content of Prefix-Based Multicast Address.
+     *
+     * @param[in]  aPrefix A pointer to a IPv6 Prefix.
+     *
+     */
+    void SetMulticastNetworkPrefix(const otIp6Prefix *aPrefix)
```

Comment:
use alias?
```suggestion
    void SetMulticastNetworkPrefix(const otIp6Prefix &aPrefix)
```

Reply:
Looking at `SetMulticastNetworkPrefix` alone, it does not actually accept `aPrefix == NULL`. So, should we prefer reference to pointer in this case?

Reply:
updated to use reference as the suggestion.